flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/client/HiveShimV310.java

```diff
@@ -239,4 +241,54 @@ public Writable hivePrimitiveToWritable(Object value) {
 		}
 		throw new FlinkHiveException("Unsupported primitive java value of class " + value.getClass().getName());
 	}
+
+	@Override
+	public void createTableWithConstraints(IMetaStoreClient client, Table table, Configuration conf,
```

Comment:
Too many parameters should be one parameter one line, both call and declaration are like this.